material-overrides/assets/stylesheets/polkadot.css

```
@@ -1523,21 +1555,27 @@ div.mermaid {
 }
 
 /* Edit this page styling */
-.actions {
+.md-content__button.md-icon {
   display: flex;
-  justify-content: center;
-  margin-top: 4em;
+  align-items: center;
+  gap: 8px;
+  text-decoration: none;
+  color: var(--storm-gray);
+  transition: color 0.3s;
+  margin: 0 auto;
+  margin-top: 0.75em;
```

```suggestion
  margin: 0.75em auto 0;
```

Just to clean it up a little bit